cpp/doxygen/developer_guide/DEVELOPER_GUIDE.md

```diff
@@ -181,11 +179,9 @@ detail see the [`libcudf++` presentation.](https://docs.google.com/presentation/
 libcudf functions typically take views as input (`column_view` or `table_view`)
 and produce `unique_ptr`s to owning objects as output. For example,
 
-```c++
-std::unique_ptr<table> sort(table_view const& input);
-```
```

Comment:
Is there a way to preserve the cpp code markdown for github view and ignore it when generating docs with Doxygen?

Not a big issue but those changes will make the doc less readable from Github

Reply:
OK I was able to find a reasonable solution here. I've configured doxygen to preprocess MD files and replace the Github-specific code fences with ones that doxygen can interpret. I can't get this to work nicely in the `DOCUMENTATION.md` file because no matter what I do doxygen tries to interpret doxygen commands inside code fences in MD files, but it works for `DEVELOPER_GUIDE.md` and `TESTING.md` which I think should address the vast majority of cases.

Reply:
> If you have any idea if there's a way to change that behavior selectively

Not yet, but we will face the same issue in cuco. Will come back if we find a solution there.

> The other files have nice highlighting now

:fire: I wasn't even sure how to do it when asking the question. Thanks for making it work! Great job!